apps/docs/content/components/collections/Listbox.mdx

```diff
@@ -1,48 +1,182 @@
 
-### Accessibility ?
+    ### Accessibility ?
 
-TODO: If we have some guidelines about this component and accessibility
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
 ## Anatomy
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+<FeatureFlag flag="rc">
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-### Concepts
+    ### Concepts
 
-TODO: links to related concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
 ### Composed Components
 
-TODO: links to related components (most likely all component which are used by the slots)
+- [Divider](./Divider)
+- [ListBoxItem](./ListBoxItem)
+
+#### Header
+
+The Header component represents a `header` within a Hopper container.
+
+#### Section
+
+The Section component represents a `section` within a Hopper container.
 
 ## Examples
 
-## Advanced customization
+### Empty state
+
+A ListBox can have an empty state.
+
+<Example src="ListBox/docs/empty" />
+
+### Disabled
+
+A ListBox with a disabled item.
+
+<Example src="ListBox/docs/disabled" />
+
+### Invalid
+
+A ListBox that is invalid.
+
+<Example src="ListBox/docs/invalid" />
+
+### Fluid
+
+ListBox that is fluid, meaning the width is 100% of its container.
+
+<Example src="ListBox/docs/fluid" />
+
+### Description
+
+ListBox items with a description.
+
+<Example src="ListBox/docs/description" />
+
+### Divider
+
+Dividers can be added to a ListBox
+
+<Example src="ListBox/docs/divider" />
+
+### Section
+
+A Listbox can have sections and section headers.
+
+<Example src="ListBox/docs/section" />
+
+### Count
+
+A ListBox can contain a count using a badge.
+
+<Example src="ListBox/docs/count" />
+
+### Dynamic Lists
+
+Items and sections can be populated from a hierarchial data structure.
+If a section has a header, the `Collection` component can be used to render the child items.
+
+<Example src="ListBox/docs/dynamicLists" />
+
+### Icons
+
+A ListBox can contain icons.
+
+<Example src="ListBox/docs/icons" />
+
+### End Icons
+
+A ListBox can contain icons at the end of a list item.
+
+<Example src="ListBox/docs/endIcons" />
+
+### Loading
+
+A ListBox can have a loading state.
+
+<Example src="ListBox/docs/loading" />
+
+### Load on scroll
+
+A ListBox can be used to load more items when scrolling.
+
+<Example src="ListBox/docs/loadOnScroll" />
+
+### Single selection
+
+A ListBox can have a single selected item.
+
+<Example src="ListBox/docs/selection" />
+
+### Single selection indicator
+
+A ListBox can have a different selection indicator for single select.
+By default, `disallowEmptySelection` is set to true when the selection indicator is a radio button.
+
+<Example src="ListBox/docs/selectionIndicator" />
+
+### Multiple selection
+
+A ListBox with multiple selected items.
+
+<Example src="ListBox/docs/multipleSelected" />
+
+### Size
```

Comment:
On the ones I've been doing I would put the title as "Sizes".
I'm wondering which one is best.. we should try and be consistent for this.